src/components/flowchart/index.js

```diff
@@ -433,12 +434,14 @@ export class FlowChart extends Component {
    * Render React elements
    */
   render() {
-    const { chartSize, layers } = this.props;
+    const { chartSize, isLoading, layers } = this.props;
     const { outerWidth = 0, outerHeight = 0 } = chartSize;
 
     return (
       <div
-        className="pipeline-flowchart kedro"
+        className={classnames('pipeline-flowchart kedro', {
```

Comment:
Every time this class is added then removed, there is potential for creating a CPU spike, which will create jank for the graph animation, so I'd suggest to instead apply the opacity directly to a SVG `g` element instead (such as the zoom wrapper).

Reply:
Ok cool, I've moved it to the zoom wrapper